dydx/client.py

```diff
@@ -146,38 +146,45 @@ def _operate(
             'ref': ref,
             'value': wei
         }
+        actualWithdrawOrDepositAddress = (
```

Comment:
Does this actually change anything? Isn’t MARKET_WETH = 0?

Reply:
doesn't change any logic, no